tests/pipelines/stable_diffusion/test_stable_diffusion_adapter.py

```diff
@@ -172,6 +203,217 @@ def test_stable_diffusion_adapter_default_case(self):
         assert np.abs(image_slice.flatten() - expected_slice).max() < 5e-3
 
 
+class StableDiffusionMultiAdapterPipelineFastTests(AdapterTests, PipelineTesterMixin, unittest.TestCase):
+    def get_dummy_components(self):
+        return super().get_dummy_components("multi_adapter")
+
+    def get_dummy_inputs(self, device, seed=0):
+        return super().get_dummy_inputs(device, seed, num_images=2)
+
+    def test_stable_diffusion_adapter_default_case(self):
+        device = "cpu"  # ensure determinism for the device-dependent torch.Generator
+        components = self.get_dummy_components()
+        sd_pipe = StableDiffusionAdapterPipeline(**components)
+        sd_pipe = sd_pipe.to(device)
+        sd_pipe.set_progress_bar_config(disable=None)
+
+        inputs = self.get_dummy_inputs(device)
+        image = sd_pipe(**inputs).images
+        image_slice = image[0, -3:, -3:, -1]
+
+        assert image.shape == (1, 64, 64, 3)
+        expected_slice = np.array([0.4902, 0.5539, 0.4317, 0.4682, 0.6190, 0.4351, 0.5018, 0.5046, 0.4772])
+        assert np.abs(image_slice.flatten() - expected_slice).max() < 5e-3
+
+    def test_inference_batch_consistent(
```

Comment:
Unfortunately, I believe this is the best way currently to correctly batch the list of images in the adapter test.

i.e. [image, image] -> [[image], [image]] instead of [image, image] -> [[image, image]]

Modifying the original mixin will touch too many existing tests

Reply:
Ok for me!